src/flate/reader.cr

```diff
@@ -130,6 +140,14 @@ class Flate::Reader < IO
     @io.close if @sync_close
   end
 
+  def unbuffered_rewind
+    raise IO::Error.new "Closed stream" if closed?
```

Comment:
`check_open`.

Reply:
Thanks! I updated the code.

Reply:
Why not just use `check_open` call like @RX14 suggested?